code/modules/projectiles/ammunition/energy/special.dm

```diff
@@ -38,6 +38,17 @@
 	select_name = "netting"
 	pellets = 6
 	variance = 40
+	var/obj/item/gun/energy/e_gun/dragnet/D
+
+/obj/item/ammo_casing/energy/net/Initialize()
+	. = ..()
+	D = loc
+	if(!istype(D))

```

Comment:
Uh let's not do this, pass the gun through creation and do not make this unmakable by non guns

Reply:
suggested change was made, but the code to do so is elsewhere. this review can be ignored

Reply:
@kevinz000 this is not the projectile.